### PR TITLE
Fixing second part of the issue #312

### PR DESCRIPTION
**Pull request by MatheusFerraz**
Periods were not translated. Fixed.
Unit tests also created.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The version numbers below try to follow the conventions at http://semver.org/.
 
 ## Unreleased
 
+- Translation of the process period in the new repositories page
+
```
Comment (rafamanzo) [inline CHANGELOG.md]: This blank line is not necessary. Can you amend to fix it?

**Comment by rafamanzo** [comment]
Welcome! Thanks for the contribution.

Your fix is indeed correct in spirit, but Rails has some API to deal with that in a similar, but more generic, form: http://guides.rubyonrails.org/v4.2/i18n.html. What do you think about it?

**Comment by MatheusFerraz** [comment]
You are right. The i18n API is the best option because of its standard solution. I used this API,  as you suggested, to provide a better solution.

File: app/helpers/repository_helper.rb [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+
+    if(language == portuguese_regex)
+      portuguese_periodicity_options
+    else
+      [["Not Periodically", 0], ["1 day", 1], ["2 days", 2], ["Weekly", 7], ["Biweekly", 15], ["Monthly", 30]]
+    end
+  end
+
+  def portuguese_periodicity_options
+    [["Sem periodicidade", 0], ["1 dia", 1], ["2 dias", 2], ["Semanal", 7], ["Quinzenal", 15], ["Mensal", 30]]
```
Comment (rafamanzo) [inline app/helpers/repository_helper.rb]: What do you think about creating those into I18n translation keys? So you don't need to use implement this hardcoded method for pt and this translation gets generic for any future locale.

Comment (MatheusFerraz) [inline app/helpers/repository_helper.rb]: Ok, I will improve the implementation.

**Comment by rafamanzo** [comment]
Now it is perfect :gem: 

Could you just add a CHANGELOG entry for this change?

Then we are ready to merge.

**Comment by MatheusFerraz** [comment]
I added a CHANGELOG entry for this change.

**Pull request merged**